site/components/pages/Home/index.module.css

```
@@ -107,6 +107,17 @@
   gap: 1.6rem;
 }
 
+.lowerStack {
+  display: flex;
+  gap: 1rem;
+}
+
+.lowerIconButton {
+  composes: iconButton;
+  gap: 0;
+  padding: 1rem;
```

Looks like this makes the button 44px tall, but the others are 40px.
If we stick to the 8px grid and use 0.8rem padding here, then the button height is 40px 👍 
```suggestion
  padding: 0.8rem;
```